docs/ambassador.md

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
This file is the same as the existing README under ambassador as https://github.com/kubernetes-sigs/kubespray/blob/5e22574402d86a7087fb146fc6f7b32c8fa80088/roles/kubernetes-apps/ingress_controller/ambassador/README.md
Do we need to add this?

Reply:
You are right, maybe I should improve this document with an example of how to use Ambassador...